src/transformers/benchmark/from_pretrained_benchmark.py

```diff
@@ -0,0 +1,32 @@
+import json
+
+from benchmark_utils_generic import BenchMark, SpeedBenchMark
+
+
+class FromPretrainedBenchMark(BenchMark):
+    def target(self, model_class, repo_id):
+        def target():
+            _ = model_class.from_pretrained(repo_id)
```

Comment:
we need to test with device map, without, with fast init, without. I think having kwargs would be nice

Reply:
With these kwargs - how are we wanting to test i.e. turning each individual feature on then off, or should there be some sort of combination? e.g. device_map with fast init

We might need to define some way of combining test features